history.c: Patch to solve the 'undefined reference to gp_read_history'

Signed-off-by: Michael Fischer <mf@go-sys.de>

diff -purN gnuplot-5.4.0.org/src/history.c gnuplot-5.4.0/src/history.c
--- gnuplot-5.4.0.org/src/history.c	2019-12-10 07:22:32.000000000 +0100
+++ gnuplot-5.4.0/src/history.c	2020-09-14 10:07:36.525441702 +0200
@@ -91,7 +91,9 @@ write_history(char *filename)
 void
 read_history(char *filename)
 {
-    gp_read_history(filename);
+#ifdef GNUPLOT_HISTORY
+    	gp_read_history(filename);
+#endif	
 }
 
